### ISSUE TITLE
Support literal `{{`, `{#`, `{%` tags in markdown code blocks / inline code

### ISSUE THREAD

**Issue by olets** [11ty/eleventy-base-blog#115]
It is not possible to use `{{` or `{#` in code blocks or inline code in markdown files without Eleventy trying to parse it. For example you can't say

```markdown
In zsh, `${#var} is the length of `var` in words (arrays) or bytes
```

or

<pre><code>To set a variable in Twig, use `set`:
```twig
{% set var = 2 %}
```</code></pre>

(My use case: shell script examples and Twig examples in blog posts.)

Disabling pre-processing of Markdown files enables `{#`

```js
// .eleventy.js

// Pre-process *.md files with: (default: `liquid`)
// markdownTemplateEngine: "njk", <-- commented out
```

Not a great solution if it worked, but it doesn't. Now

```
`{#`
```

will be treated literally, and compile as `<code>{#</code>`

but

```
`{{`
```

and

```
`{%`
```

will try to compile as Liquid.

The fundamental limitation here may be something about markdown-it, or something about Eleventy's data flow. I ran into it in this demo, lmk if I should move the issue to a different repo.

**Comment by pdehaan** [comment]
Correct, by default Eleventy will try and prerender the Markdown templates w/ LiquidJS.
We recently discussed it in https://github.com/11ty/eleventy/discussions/2148, which includes a couple potential solutions based on your requirements.

**Comment by olets** [comment]
Thanks for the link!
